tests/test_integration.py

```diff
@@ -4,13 +4,36 @@
 import yaml
 
 
-def test_schemas():
-    schemas_root = Path(__file__).parent.parent / "resources" / "stsci.edu" / "schemas"
+def test_resources():
+    resources_root = Path(__file__).parent.parent / "resources"
```

Comment:
I have to admit my ignorance of this new paradigm. I'm not sure I like the magical use of the / operator though. Maybe I will come to love it though.

Reply:
It's a little cutesy, isn't it...